src/custom-apps/components/WebhookSubscriptionQuery/WebhookSubscriptionQuery.test.tsx

```diff
@@ -30,6 +28,22 @@ jest.mock("@saleor/macaw-ui", () => ({
   DialogHeader: jest.fn(() => () => <></>),
 }));
 
+jest.mock("@saleor/macaw-ui/next", () => ({
+  useTheme: jest.fn(() => ({
```

Comment:
**suggestion**: I'm wondering if wrapping test in `ThemeProvider` will work here instead of mocking it here? 🤔